CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.12.0] - 2020-12-17
+### Added
+- numba-dppy is a standalone package now.
+- Offload diagnostics.
+- Controllable fallback.
+- Add flags to generate debug symbols.
+- Implementation of `np.linalg.eig`.
+
+### Changed
+- Rename `DPPL` to `DPPY`.
+- `DPPLTestCase` replaced with `unittest.TestCase`.
+- All tests and examples use `with device_context`.
```

Comment:
- The `dpnp` integration to support NumPy functions on SYCL devices now uses Numba's `@overload` functionality.

Reply:
Done